diff --git a/arch/arm64/kernel/cpuinfo.c b/arch/arm64/kernel/cpuinfo.c
index 690ef947a22b..e5b4c3b4cb5f 100644
--- a/arch/arm64/kernel/cpuinfo.c
+++ b/arch/arm64/kernel/cpuinfo.c
@@ -140,7 +140,7 @@ static int c_show(struct seq_file *m, void *v)
 		{ "S905X2", MESON_CPU_MAJOR_ID_G12A, 0x40, 0xf0 },
 		{ "S922X", MESON_CPU_MAJOR_ID_G12B, 0x40, 0xf0 },
 		{ "A311D", MESON_CPU_MAJOR_ID_G12B, 0x10, 0xf0 },
-		{ "S905X3", MESON_CPU_MAJOR_ID_SM1, 0x5, 0xf },
+		{ "S905X3", MESON_CPU_MAJOR_ID_SM1, 0x10, 0xf0 },
 		{ "S905D3", MESON_CPU_MAJOR_ID_SM1, 0xb0, 0xf0 },
 	};
 #endif
